src/panoramic/cli/data_connections.py

```diff
@@ -0,0 +1,155 @@
+        'user': user,
+        'password': password,
+        'host': host,
+        'port': port,
+        'database_name': database_name,
+    }
+
+    if not no_test:
+        ok, error = DataConnections.test(data_connections[name])
+        if not ok:
+            print(f'Failed to create data connection: {error}')
+            return
+
+    DataConnections.save(data_connections)
+    print('Data connection was successfully created!')
+
+
+def update_data_connection_command(name, type, user, host, port, password, password_stdin, database_name, no_test):
+    """CLI command. Update specific data connection."""
+    data_connections = DataConnections.load()
+    if name not in data_connections:
+        raise click.ClickException(f'Data connection with name "{name}" not found.')
+
+    if password_stdin:
+        password = click.prompt('Enter password: ', hide_input=True, type=str)
+
+    if type:
+        data_connections[name]['type'] = type
+    if user:
+        data_connections[name]['user'] = user
+    if password:
+        data_connections[name]['password'] = password
+    if host:
+        data_connections[name]['host'] = host
+    if port:
+        data_connections[name]['port'] = port
+    if database_name:
+        data_connections[name]['database_name'] = database_name
+
+    if not no_test:
+        ok, error = DataConnections.test(data_connections[name])
+        if not ok:
+            print(f'Failed to create data connection: {error}')
+            return
+
+    DataConnections.save(data_connections)
+    print('Data connection was successfully created!')
+
+
+def list_data_connections_command(show_password):
+    """CLI command. List all data connections."""
+    data_connections = DataConnections.load()
+    if not data_connections:
+        config_file = Paths.config_file()
+        print(
+            f'No data connections found.\n'
+            f'Use "pano data-connections create" to create data connection or edit "{config_file}" file.'
+        )
+        exit(0)
+
+    for name, connection in data_connections.items():
+        connection_string = DataConnections.create_connection_string(connection)
+        if not show_password:
+            connection_string = connection_string.replace(connection['password'], '*****')
+        print(f'{name}: {connection_string}')
+
+
+def remove_data_connection_command(name):
+    """CLI command. Remove existing data connection."""
+    data_connections = DataConnections.load()
+    if name not in data_connections:
+        raise click.ClickException(f'Data connection with name "{name}" not found.')
+
+    del data_connections[name]
+    DataConnections.save(data_connections)
+
+
+def test_all_data_connections_command():
+    """CLI command. Test all data connections by trying to connect to the database."""
+    for name, connection in DataConnections.load().items():
+        ok, error = DataConnections.test(connection)
+        if ok:
+            print(f'{name}... OK')
```

Comment:
I think we should use `echo_info` instead of `print`

Reply:
replaced print for echo_info